packages/react/src/components/Table/TableHeader.tsx

```diff
@@ -11,45 +12,69 @@ interface TableHeaderProps extends ThHTMLAttributes<HTMLTableCellElement> {
   sortDirection?: SortDirection;
   onSort?: () => void;
   className?: string;
+  sortAscendingAnnouncement?: string;
+  sortDescendingAnnoucnement?: string;
```

Comment:
There's a typo here in `sortDescendingAnnoucnement`

Reply:
spelling is hard